docs/git-cola.rst

```diff
@@ -762,6 +762,12 @@ Defaults to `true`.  This setting is configured using the `Preferences`
 dialog, but it can be toggled for one-off usage using the commit message
 editor's options sub-menu.
 
+cola.logdate
+--------------
```

Comment:
Make sure that the `---------` line is the exact same length as `cola.logdate` otherwise sphinx will complain about it.